data/BlueprintApi.js

```diff
@@ -1,88 +1,48 @@
-import constants from "../core/constants";
 import NotificationsApi from "./NotificationsApi";
-import utils from "../core/utils";
+import * as composer from "../core/composer";
 
 class BlueprintApi {
   constructor() {
     this.blueprint = undefined;
   }
 
   handleCommitBlueprint(blueprint) {
-    const p = new Promise((resolve, reject) => {
-      this.postBlueprint(this.postedBlueprintData(blueprint))
-        .then(() => {
-          NotificationsApi.closeNotification(undefined, "committing");
-          NotificationsApi.displayNotification(blueprint.name, "committed");
-          resolve();
-        })
-        .catch(e => {
-          console.log(`Error committing blueprint: ${e}`);
-          NotificationsApi.closeNotification(undefined, "committing");
-          NotificationsApi.displayNotification(blueprint.name, "commitFailed");
-          reject();
-        });
-    });
-    return p;
-  }
-
-  postedBlueprintData(blueprint) {
```

Comment:
This is still being used in sagas/blueprints.js, (e.g. `BlueprintApi.postedBlueprintData`).

Reply:
Oops. Thanks for the catch. I've uploaded a fix.